ODIN_II/SRC/read_blif.cpp

```diff
@@ -1556,10 +1556,13 @@ hard_block_ports *get_hard_block_ports(char **pins, int count)
 			ports->sizes[ports->count] = 0;
 			ports->names[ports->count] = portname;
 			ports->count++;
+			prev_portname = portname;
+
```

Comment:
This changes the behavior. Only gets assigned on I == 0 or when previous is equal to current

Reply:
Can you initialize prev_portname at the beginning of the function to NULL rather than 0